README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+Hey there!  My name is Kristina Wald.  Here are some quick facts about me:
+
+# Age
+25
+
+# Where I'm from
+Hyde Park - *grew up practically right on campus!*
+
+# What I'm studying
+Behavioral Science at Booth
+
+# Research interests
+* Person perception
+* Impression formation
+* Social interaction
+
+# Favorite things to do
+* Take walks *(especially along Lake Michigan!)*
+* Watch _**The Bachelor**_
+* Sleep
+
+# Fun Facts
+1. I go to Starbucks every day, but **don't really drink coffee!**
+1. Visiting Antarctica is on my bucket list.
+1. I lived in New York for 6 years and _**fell madly in love with it!**_
+
+# LinkedIn Page
+[Click here](https://www.linkedin.com/in/kristina-wald-77a4a763/)
+
+# Picture of Me
+![Me](Wald-2644.jpg)
+
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |       X    |              |            |

Overall it's a very great post and I'm glad to know more about you!